src/styles/blocks/page.css

```diff
@@ -0,0 +1,3 @@
+.page__search-bar {
+  margin-top: 20px;
```

Comment:
it is not good practice when margin of the child element is out of parent element, if you need to add space between parent and child borders it is better to use padding for parent, also when you fix it you will not need this class `page__search-bar` any more